src/widgets/mentors-wanted/ui/mentors-wanted.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { screen } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import { MentorsWanted } from './mentors-wanted';
+import { LINKS } from '@/app/const';
+import { renderWithRouter } from '@/shared/__tests__/utils';
+import mentorImg from '@/shared/assets/mentors-wanted-poster.webp';
+
+const mockedData = {
+  title: 'Mentors Wanted!',
+  paragraph: /If you are interested in mentoring our students/i,
+  link: LINKS.ANGULAR_MENTORING,
+  image: mentorImg,
+  alt: 'Sloth - mascot dresses as a detective',
+};
+
+describe('MentorsWanted component', () => {
+  beforeEach(() => {
+    renderWithRouter(<MentorsWanted />);
+  });
+
+  it('renders without crashing', () => {
+    const mentorsWanted = screen.getByTestId('mentors-wanted');
+
+    expect(mentorsWanted).toBeVisible();
+  });
```

Comment:
Tests like this seem redundant to me because if something goes wrong, the test below will also fail

Reply:
I added it because we have the same test in some other components. I can move it to the test below

Reply:
Yes, please combine them

Reply:
Done